github/event_types.go

```diff
@@ -130,17 +130,6 @@ type GollumEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
-// EditChange represents the changes when an issue, pull request, or comment has
-// been edited.
-type EditChange struct {
-	Title *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"title,omitempty"`
-	Body *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"body,omitempty"`
-}
```

Comment:
When I made https://github.com/google/go-github/pull/544#discussion_r102259209, I didn't mean for you to actually remove `EditChange` and modify the other existing structs, I was referring only to the 3 new event types being added for #530.

However, given you've already done it, and this is probably better in the long term, we should consider keeping it. @gmlewis, what do you think about this?

It shouldn't be a breaking API change for most uses that just access the fields ala `something.Title.From`, but it might still be a breaking change for code that tries to copy or manipulate these structs.